setup.py

```diff
@@ -26,7 +26,7 @@ def read(fname):
 
 setup(
     name='sagemaker_pytorch_serving_container',
-    version='1.2',
+    version='1.3',
```

Comment:
Please have this change approved by someone from the SageMaker team, at least just by asking them.

We need to know that this is the only place where version needs to be updated from 1.2 to 1.3